### DISCUSSION TITLE
希望logo图标能分白天和黑夜模式展示

### DISCUSSION THREAD

**Discussion by yuhuan1126** [AlistGo/alist#7193] [Ideas]
如题，希望可以白天模式是一个logo图标，夜晚模式是一个logo图标，如果图标背景透明的话基本只能兼顾一个模式下的啊

**Comment by anwen-anyi** [comment]
可以自己配置两个logo，上面的是白天，第二行的是夜间
![image](https://github.com/user-attachments/assets/b3ba23bb-ec67-4deb-a0fd-642f8612a057)
